kong/timing/context.lua

```diff
@@ -75,11 +74,44 @@ function _M:set_root_context_prop(k, v)
 end
 
 
+function _M:finalize(subcontext)
```

Comment:
I just wonder why not always use nanosecond resolution, but only use it if greater than 1?

For example, if the value is 1.010ms, the extra 10ns means a lot.

Reply:
This is initially for a more clean result. But this limitation should be removed.

Reply:
done